### ISSUE TITLE
 implementation in Domoticz

### ISSUE THREAD

**Issue by codystoica** [rytilahti/python-miio#1733]
Hello!

**Is your feature request related to a problem? Please describe.**
I did not find any plugin developed for this model Xiaomi Vacuum 2 Pro+ "dreame.vacuum.p2041o" with implementation in Domoticz.

**Describe the solution you'd like**
you can adapt this plugin for Domoticz, I noticed that the phython-miio library has support for this model "dreame.vacuum.p2041o".
I aim to be able to turn on the light bulbs automatically in the room where it is vacuuming at night, this would be possible if it reported the room_id in which it is vacuuming.

I would be very grateful

**Device information:**
If the enhancement is device-specific, please include also the following information.

  - Name(s) of the device: Xiaomi Vacuum 2 Pro+
  - Link: https://dontvacuum.me/robotinfo/detail_dreame.vacuum.p2041o_0.html

Use `miiocli device --ip <ip address> --token <token> info`.

  - Model: dreame.vacuum.p2041o
  - Hardware version: Linux /4.1.2_1107 
  - Firmware version:  4.3.3_1111

**Comment by rytilahti** [comment]
I don't think this device reports the that information, you can see https://github.com/rytilahti/python-miio/blob/master/miio/integrations/dreame/vacuum/dreamevacuum_miot.py#L77-L122 for the full list of sensors, settings, and actions of the device.

Regarding to domoticz support, I don't know anything about that and it's best to ask them about that.
